src/test/java/com/mathworks/ci/RunMatlabTestsStepTest.java

```diff
@@ -67,15 +67,15 @@ public void verifyOnslave() throws Exception {
     }
 
     /*
-     * Verify artifact path is correct.
+     * Verify artifact path is correct. Need to move this to integration test.
      */
 
-    @Test
+    
     public void verifyArtifactPath() throws Exception {
```

Comment:
This test didn't run in the current qualification. Need to add the @ test annotation.

Reply:
Same as earlier.